source/nodejs/adaptivecards-designer/src/adaptivecards-designer.css

```diff
@@ -196,7 +196,7 @@
 }
 
 .acd-dialog-frame {
-    background-color: white;
```

Comment:
Sorry I just noticed this - looks we reverted a change from this PR here - https://github.com/microsoft/AdaptiveCards/pull/7497.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/98650930/190459681-737cdcea-5e8d-45ae-b46e-8fd442290490.png">

Reply:
Yes - #7122 fixed the background in high contrast mode, but broke it in standard view.

<img width="986" alt="image" src="https://user-images.githubusercontent.com/98650930/190461111-fd4ca168-5dbb-457a-ae86-b1b10e5ccac9.png">

#7497 Addressed the issue and only applied `ActiveBorder` when in high contrast.

Reply:
reverted the activateboard on main branch and this branch